File: prime-number/perl/cmd.pl
my $primeNumberCount = 100;
if (scalar(@ARGV) == 1) {
    $primeNumberCount = int(@ARGV[0]);
}

my $number = 0;
while ($primeNumberCount > 0) {
    $number++;

    my $j = 0;
    for (my $i = 1; $i <= $number; $i++){
        if ($number % $i == 0) {
            $j++;
        }
    }

    if ($j == 2) {
        $primeNumberCount--;
    }
}

print "The latest prime number: $number\n";

File: prime-number/python/cmd.py
import sys

primeNumberCount = int(sys.argv[1]) if (len(sys.argv) > 1) else 100

number = 0
while (primeNumberCount > 0):
    number += 1
    j = 0

    for i in range(1, number + 1):
        if (number % i == 0):
            j += 1

    if (j == 2):
        primeNumberCount -= 1

print ('The latest prime number: ', number)
